### DISCUSSION TITLE
[Script request] Ghost CMS

### DISCUSSION THREAD

**Discussion by kiwinh0** [community-scripts/ProxmoxVE#167] [Request script]
### Application Name

Ghost

### Website

https://ghost.org/

### Description

Ghost is a powerful app for professional publishers to create, share, and grow a business around their content. It comes with modern tools to build a website, publish content, send newsletters & offer paid subscriptions to members.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
merged
